doc/advanced-configuration/outgoing-connections.md

```diff
@@ -190,37 +190,38 @@ sslrootcert = /path/to/ca/cert
 
 ## HTTP connections setup
 
-Some MongooseIM modules need an HTTP connection to external service.
-These pools need to be configured and started before the module needs them.
+Some MongooseIM modules need an HTTP connection to an external service.
+These pools need to be configured and started before the module that needs them.
 Below is a sample configuration:
 
 ```erlang
 {outgoing_pools, [
  {http, global, default, PoolOptions, ConnectionOptions}
 ]}.
 ```
-
-`PoolOptions` are described above, below there are the recommended `PoolOptions` for `HTTP` pools:
+where `PoolOptions` is as previously described.
+Recommended `PoolOptions` for `HTTP` pools are:
 
 * `strategy` - the recommended value is `available_worker`
 * `call_timeout` - it should be equal or longer than the value set in `request_timeout` below.
 
 `ConnectionOptions` can take the following `{key, value}` pairs:
 
-* `{server, HostName}` - string, default: `"http://localhost"` - the URL of the destination HTTP server (including a port number if needed).
-* `{path_prefix, Prefix}` - string, default: `"/"` - the part of the destination URL that is appended to the host name (`host` option).
+* `{server, HostName}` - string - the URL of the destination HTTP server (including a port number if needed).
+* `{path_prefix, Prefix}` - string, default: `"/"` - the part of the destination URL that is appended to the host name (`Host` option).
 * `{request_timeout, TimeoutValue}` - non-negative integer, default: `2000` - maximum number of milliseconds to wait for the HTTP response.
-* `{http_opts, HTTPOptions}` - list, default: `[]` - can be used to pass extra parameters which are passed to [fusco], the library used for making the HTTP calls.
-  More details about the possible `http_opts` can be found in [fusco]'s documentation.
+* `{http_opts, HTTPOptions}` - extra parameters to the library responsible for making the HTTP calls.
```

Comment:
In this file, there is an example below, how to set certification validation for the `HTTP` workers. Does the example still work with `gun`? It was made and checked with `fusco`.